template/build/webpack.base.conf.js

```diff
@@ -16,10 +16,12 @@ module.exports = {
     extensions: ['', '.js', '.vue'],
     fallback: [path.join(__dirname, '../node_modules')],
     alias: {
-      {{#if_eq build "standalone"}}
-      // nessessary to make "import Vue from 'vue'" load the standalone version.
+      {{#if_eq build "standalone"}}∞
       'vue': 'vue/dist/vue',
       {{/if_eq}}
```

Comment:
They will, technically. I thought I leave them in to make things clear and not take any chances of it exporting the runtime-only for some people depending on when all the 2.0 goes live in non and reaches the users. 

But it's not nessessary.